### ISSUE TITLE
spell check gives warning, treated as error

### ISSUE THREAD

**Issue by kevinsung** [Qiskit/qiskit-nature#469]
### Environment

- **Qiskit Nature version**: dcdd1ddc6da85cbecaeae9b3689af1548fd1e8aa
- **Python version**: 3.9
- **Operating system**: RHEL 8.5

### What is happening?

The spell check gives a warning, which is by default treated as an error.

### How can we reproduce the issue?

1. Create a fresh virtual environment
2. Install qiskit-nature from source, along with the dev tools.
3. Run `make spell`.

Result:
Warning, treated as error:
```
...
executing qiskit_nature.operators.second_quantization.FermionicOpantization.FermionicOp                                                                                             
executing qiskit_nature.operators.second_quantization.SpinOpnd_quantization.SpinOp                                                                                                  
executing qiskit_nature.problems.second_quantization.lattice.HyperCubicLatticetice.HyperCubicLattice                                                                                
reading sources... [ 98%] tutorials/01_electronic_structure                                                                                                                         
Warning, treated as error:
/home/kevinsung/projects/qiskit-nature/qiskit_nature/runtime/vqe_program.py:docstring of qiskit_nature.runtime.vqe_program.VQEProgram:8:Unexpected indentation.
make: *** [Makefile:64: spell] Error 2
```

### What should happen?

No warning and no error.

**Comment by mrossinek** [comment]
Hi @kevinsung! I was able to reproduce this but subsequently able to fix it be ensuring two things:
- having the latest updates of the requirements installed:
```
pip install -r requirements.txt -r requirements-dev.txt -c constraints.txt --update
```
- running `make clean_sphinx` to ensure the sphinx cache is empty

Can you please check if that fixes things for you?
If yes, you will likely see one more spelling error related to the word "onsite" which will also get fixed by #470.

**Comment by kevinsung** [comment]
Thanks, that fixed it!
